data/mods/Magiclysm/monsters/Ogre.json

```
@@ -0,0 +1,29 @@
+[
+  {
+    "id": "mon_ogre",
+    "type": "MONSTER",
+    "name": { "str": "Ogre" },
+    "description": "Green beast known for living in swamps and eating humans, it will kill anything it gets its hands on.",
+    "default_faction": "ogres",
+    "bodytype": "human",
+    "species": [ "OGRE" ],
+    "volume": "100 L",
+    "weight": "130 kg",
+    "hp": 140,
+    "speed": 123,
+    "material": [ "flesh" ],
+    "symbol": "O",
+    "color": "green",
+    "aggression": 56,
+    "morale": 113,
+    "melee_skill": 7,
+    "melee_dice": 5,
+    "melee_dice_sides": 8,
+    "dodge": 20,
```

```suggestion
    "dodge": 3,
```
From their size, that's being generous.